server/plugin/command/create_event.go

```diff
@@ -12,10 +13,15 @@ import (
 
 func getCreateEventFlagSet() *flag.FlagSet {
 	flagSet := flag.NewFlagSet("create", flag.ContinueOnError)
-	flagSet.String("subject", "", "Subject of the Event (no spaces for now)")
+	flagSet.Bool("help", false, "show help")
+	flagSet.String("subject", "", "Subject of the event (no spaces for now)")
```

Comment:
Since subject and body are not practical to pass in as arguments, I recommend recommend naming these flags `--test-subject` etc. to be more intention revealing. (also `...body, location`)

Reply:
That's fine.  I'll prepend `test-` to the `location`, `body`, `subject` flags.  Note that I have added a jira ticket to address the `no spaces` issue.

https://mattermost.atlassian.net/browse/MM-21347

Reply:
The `location` flag does work.  I considered the tradeoff between requiring one flag `location` with the comma separated list of values, versus splitting `location` into several flags.  We could introduce `--location.street`, `--location.city`, `--location.state`, .etc as a alternative method.  the tradeoff being that a user would be required to list many extra flags.

Ticket created:
https://mattermost.atlassian.net/browse/MM-21390